tests/Sniffs/Functions/FunctionsFixture.php

```diff
@@ -135,7 +135,7 @@ public function missingArgHintThree(string $arg1, $arg2): string {
 	}
 
 	// Next line should warn about unused type hint
-	public function hasReturnHintButNoReturn() : string {
+	public function hasReturnHintButNoReturn(): string {
```

Comment:
Maybe we should add a test to this file that uses the colon incorrectly as well, since the Closures tests only test anonymous functions.

Reply:
I've added more tests in the FunctionsFixture.php file